bgpd/bgp_network.c

```diff
@@ -160,8 +160,12 @@ static int bgp_md5_set_password(struct peer *peer, const char *password)
 	 */
 	frr_with_privs(&bgpd_privs) {
 		for (ALL_LIST_ELEMENTS_RO(bm->listen_sockets, node, listener))
-			if (listener->su.sa.sa_family
-			    == peer->su.sa.sa_family) {
+			if (listener->su.sa.sa_family == peer->su.sa.sa_family
```

Comment:
Can't this be split into separate checks (for better readability)? I mean:

```
if (A) {
  if (B)
    continue;

  if (C)
    continue;
}
```

Absolutely non-blocking comment, but just as I mentioned, easier to read.

Reply:
done